Some very recent commit broke building on Tiger PPC

The build made it to here:
```
CompileC build/MacVim.build/Release/MacVim.build/Objects-normal/ppc/MMWindowController.o /bld/macvim/src/MacVim/MMWindowController.m normal ppc objective-c com.apple.compilers.gcc.4_0
    cd /bld/macvim/src/MacVim
    /Developer/usr/bin/gcc-4.0 -x objective-c -arch ppc -pipe -Wno-trigraphs -fobjc-exceptions -fpascal-strings -fasm-blocks -Os -Wreturn-type -Wunused-variable -fmessage-length=0 -mtune=G5 -fvisibility=hidden -I/bld/macvim/src/MacVim/build/MacVim.build/Release/MacVim.build/MacVim.hmap -mdynamic-no-pic -F/bld/macvim/src/MacVim/build/Release -F/bld/macvim/src/MacVim -I/bld/macvim/src/MacVim/build/Release/include -I/bld/macvim/src/MacVim/build/MacVim.build/Release/MacVim.build/DerivedSources -c /bld/macvim/src/MacVim/MMWindowController.m -o /bld/macvim/src/MacVim/build/MacVim.build/Release/MacVim.build/Objects-normal/ppc/MMWindowController.o
/bld/macvim/src/MacVim/MMWindowController.m: In function ‘-[MMWindowController windowDidResize:]’:
/bld/macvim/src/MacVim/MMWindowController.m:1035: error: ‘NSAppKitVersionNumber10_10_Max’ undeclared (first use in this function)
/bld/macvim/src/MacVim/MMWindowController.m:1035: error: (Each undeclared identifier is reported only once
/bld/macvim/src/MacVim/MMWindowController.m:1035: error: for each function it appears in.)
** BUILD FAILED **
Makefile:2930: recipe for target 'macvim' failed
make[1]: *** [macvim] Error 1
make[1]: Leaving directory '/bld/macvim/src'
Makefile:26: recipe for target 'first' failed
make: *** [first] Error 2
```

I haven't tested on Leopard yet; will wait until this is fixed first since it seems likely to break in the same place on Leopard too.

--- a/src/MacVim/MacVim.h
+++ b/src/MacVim/MacVim.h
@@ -345,8 +345,12 @@ typedef unsigned int NSUInteger;
 # define NSINTEGER_DEFINED 1
 #endif
 
-#ifndef NSAppKitVersionNumber10_4  // Needed for pre-10.5 SDK
+// Needed for pre-10.5 SDK
+#ifndef NSAppKitVersionNumber10_4
 # define NSAppKitVersionNumber10_4 824
+#endif
+#ifndef NSAppKitVersionNumber10_10_Max
+# define NSAppKitVersionNumber10_10_Max 1349
 #endif
 
 #ifndef CGFLOAT_DEFINED

